logging-cfg-dev.yaml

```diff
@@ -0,0 +1,81 @@
+version: 1
```

Comment:
All three logging config should be updated, compare with `service-qrcode`. In your config there is two issue that have been meanwhile  fixed in qrcode: 

1. using utc isotime
2. removed duplicate time in json output

The config in doc-guideline are still not up to date :-/ I'll update them today.